src/App/VPSScan/Scan/ScotlandYard.hs

```diff
@@ -82,17 +89,27 @@ createScotlandYardScan ScotlandYardOpts {..} = runHTTP $ do
   pure (responseBody resp)
 
 -- Given the results from a run of IPR, a scan ID and a URL for Scotland Yard,
--- post the IPR result to the "Upload Scan Data" endpoint on Scotland Yard
--- POST /scans/{scanID}/discovered_licenses
-uploadIPRResults :: (ToJSON a, Has (Lift IO) sig m, Has Diagnostics sig m) => Text -> a -> ScotlandYardOpts -> m ()
+-- post the IPR result in chunks of 1000 entries to the "Upload IPR Data" endpoint on Scotland Yard.
```

Comment:
I'd love if this change allowed us to no longer need customers to reconfigure their ingresses for large bodies.

My vague thoughts are:
* Would it be possible to have a "smart upload window" that resizes smaller if we get a 413 error?
* Would it be possible to split based on upload body size instead of number of files?

If neither of these are simple to accomplish, let's just keep going as is and make a ticket to revisit this later- either way this is better than what we have now for sure!

Reply:
That's a really good idea. Let me play around with this, timeboxed to an hour or so, and get back to you.

Reply:
I have something. It's really inefficient in that it calculates the length of the chunks many times, but it works. I think we can ship this now and refactor to make it a bit less gross later.

Reply:
Sounds great! 👍 thanks so much.